README.md

```diff
@@ -1,168 +1,222 @@
 # DSP-UI library
 
-This is the demo and developing environment for DSP-UI library (@knora/ui) composed of 4 modules.
+This is the demo and developing environment for DSP-UI-LIB library (@dasch-swiss/dsp-ui) composed of 4 modules.
```

Comment:
for DSP-UI-LIB library -> the for DSP-UI-LIB library?

Reply:
It should be DSP-UI-LIB or DSP-UI library. I think the term `lib` in `DSP-UI-LIB library` is redundant.

Reply:
https://github.com/dasch-swiss/knora-ui-ng-lib/pull/72/commits/4c823d71aad8c4b095055d87b54ebdaf886cf1a4